contrib/python-graphviz/patches/0001-nuke-the-cov-meme.patch

```diff
@@ -0,0 +1,35 @@
+From 8dc4d8a5fe292a9ad8dd050348165a101dbc379e Mon Sep 17 00:00:00 2001
+From: miko <mikoxyzzz@gmail.com>
+Date: Sat, 29 Jun 2024 18:42:02 +0200
+Subject: [PATCH] nuke the cov meme
+
+---
+ setup.cfg | 6 ------
+ 1 file changed, 6 deletions(-)
+
+diff --git a/setup.cfg b/setup.cfg
+index a575f6c..dd99809 100644
+--- a/setup.cfg
++++ b/setup.cfg
+@@ -17,18 +17,12 @@ addopts =
+ 	--doctest-continue-on-failure
+ 	-r fsxX
+ 	--durations=10
+-	--cov --cov-report=term --cov-report=html

```

Comment:
you only need to patch this and not the stuff below pretty sure so you can make the diff smaller